test/dataset_collection/test_dataset_collection.py

```diff
@@ -329,6 +344,21 @@ def test_inline_strange_body2(self):
                 open(test_filepath, encoding='utf-8') as test_ex:
             self.assertEqual(actual_file.read(), test_ex.read())
 
+    def test_inline_comments_at_the_end(self):
```

Comment:
Please add some empty lines in the method body to break it into logical blocks for better understandability.